### ISSUE TITLE
Option to enable/disable correlation based on Warninglists entries

### ISSUE THREAD

**Issue by dmaciejak** [MISP/MISP#1978]
In the event view we can see a correlate checkbox for attributes, it would be useful to have a global parameter in MISP settings to let the user enable/disable correlation for entries identified as part of the warninglists (maybe the feature exists already but i missed it).

**Comment by iglocska** [comment]
There is already a setting that allows users to disable correlations per attribute or per event (you do need to enable it in the server settings though), it's called `MISP.allow_disabling_correlation`

Or did you mean automatically disabling correlations if a warninglist entry was found?

**Comment by dmaciejak** [comment]
Yes, I meant disabling correlation with that specific attribute if it is found to be in a warninglist.
